code/game/gamemodes/clock_cult/clock_unsorted.dm

```diff
@@ -97,6 +97,97 @@
 	usr.verbs -= /mob/living/carbon/human/proc/function_call
 	return 1
 
+#define GEAR_SECURE 1 //Construction defines for the pinion airlock
+#define GEAR_UNFASTENED 2
+#define GEAR_LOOSE 3
+
+/obj/machinery/door/airlock/clockwork
```

Comment:
Add a New() for these with this so you get a cool spawn animation(this may require updating your code so it can compile)
    
var/turf/T = get_turf(src)
PoolOrNew(/obj/effect/overlay/temp/ratvar/door, T)
PoolOrNew(/obj/effect/overlay/temp/ratvar/beam/door, T)